backend/apps/volontulo/serializers.py

```diff
@@ -204,7 +209,10 @@ def get_is_administrator(obj):
     def get_organizations(self, obj):
         """Returns organizations that user belongs to."""
         qs = obj.userprofile.organizations.all()
-        return OrganizationSerializer(qs, many=True, context=self.context).data
+        return OrganizationSerializer(qs, many=True, context=self.context).data\
+
```

Comment:
I believe, that this `\` at the end of line cause linter to make some mistake here, but we should leave one empty line between methods inside class.

Travis error: `./apps/volontulo/serializers.py:214:5: E301 expected 1 blank line, found 0`

General good practices for Python formating are available in PEP-8, for blank lines see: https://www.python.org/dev/peps/pep-0008/#blank-lines